src/components/shared/Footer.tsx

```diff
@@ -1,39 +1,159 @@
-            <div>
-              <span>{Translate("Footer", ["CiteAs"])}</span> 
-              <a href={lancetId} target="__blank" rel="noopener noreferrer" className="cite-link">
-                <span className="italics">{Translate("Footer", ["LancetInfDis"])}</span> {Translate("Footer", ["Article"])}
-              </a> 
-            </div>
+  return isMobileDeviceOrTablet ? renderMobileFooter(updatedAt) : renderDesktopFooter(updatedAt)
+}
+
+const renderDesktopFooter = (updatedAt: string) => (
+  <footer className={'container-fluid mx-0'}>
+      <div className="footer-visible-section row justify-content-between d-flex align-items-center text-center">
+        <div className="col-2">
+          <UpdatedAt updatedAt={updatedAt}/>
+        </div>
+        <div className="col-2">
+            <Citation/>
+        </div>
+      </div>
+      <div className="row justify-content-center mt-3">
+        <div className="col-8">
+          <div className="row">
+            <PageLinks/>
+          </div>
+        </div>
+      </div>
+      <div className="row justify-content-center mt-5">
+        <div className="col-8">
+          <div className="sponsers-pill my-2">
+            <Sponsers/>
+          </div>
+          <WhoDisclaimer/>
         </div>
+      </div>
+      <div className="row justify-content-center py-5 text-center">
+          <Link className="px-1" to={withLocaleUrl("PrivacyPolicy")}>{Translate('PrivacyPolicy')}</Link>
+        |
+          <Link className="px-1" to={withLocaleUrl("CookiePolicy")}>{Translate('CookiePolicy')}</Link>
+        |
+          <Link className="px-1" to={withLocaleUrl("TermsOfUse")}>{Translate('TermsOfUse')}</Link>
+      </div>
     </footer>
-  )
-}
\ No newline at end of file
+)
+
+const renderMobileFooter = (updatedAt: string) => (
+  <footer className={'container-fluid mx-0'}>
+      <div className="footer-visible-section row justify-content-around d-flex align-items-center text-center">
+          <UpdatedAt updatedAt={updatedAt}/>
+          <Citation/>
+      </div>
+      <div className="row justify-content-center mt-5">
+        <div className="col-8">
+          <div className="row sponsers-pill">
+            <Sponsers/>
+          </div>
+          <WhoDisclaimer/>
+        </div>
+      </div>
+      <div className="row justify-content-center mt-5 pb-5 text-center">
+        <div>
+          <Link className="px-1" to={withLocaleUrl("PrivacyPolicy")}>{Translate('PrivacyPolicy')}</Link>
+        </div>
+        |
+        <div>
+          <Link className="px-1" to={withLocaleUrl("CookiePolicy")}>{Translate('CookiePolicy')}</Link>
+        </div>
+        |
+        <div>
+          <Link className="px-1" to={withLocaleUrl("TermsOfUse")}>{Translate('TermsOfUse')}</Link>
+        </div>
+      </div>
+    </footer>
+)
+
+const renderUpdateDate = (updatedAt: string) => {
+  return NumberDateToWordDate(updatedAt)
+}
+
+type UpdatedAtProps = {
```

Comment:
is it convention to have types / interfaces at the top? not sure but that's what i usaully see

Reply:
Yeah I agree with Prannoy on this one

Reply:
Will change!